third_party/rust/curve25519_dalek/v3/crate/Cargo.toml

```diff
@@ -62,7 +62,7 @@ version = "^2.2.1"
 default-features = false
 
 [dependencies.zeroize]
-version = ">=1, <1.4"
+version = "1"
```

Comment:
This seems wrong. We can't build against current `zeroize` until we move to curve25519_dalek v4, which is still in rc.

Reply:
the version it's building against is `1.5.5` and it builds correctly, is there a problem even if the build works?

Reply:
I'm going to resolve this for now because this doesn't actually make any changes, the dep is already on 1.5.5 in gn, but I brought it up in DM to see if this is an issue in the current code.